definitions/divorce/json/CaseEvent/CaseEvent-alternative-service-nonprod.json

```diff
@@ -33,5 +33,19 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/petition-issued?generateAosInvitation=true",
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "03/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "reissue",
+    "Name": "Reissue petition",
+    "Description": "Reissue petition",
+    "DisplayOrder": 3,
+    "PreConditionState(s)": "AosAwaiting;AosStarted;AosOverdue;Rejected;Issued;AwaitingDocuments",
```

Comment:
Just need to remove "AosOverdue" from preconditionstates 🙂. Apart form that seems good to go.

Reply:
True, done 👌🏾